roles/conjur_host_identity/tasks/clean_identity_config.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Clean conjur.identity
+  file:
+    state: absent
+    path: /etc/conjur.identity
+
+- name: Clean conjur.pem
+  file:
+    state: absent
+    path: /etc/conjur.pem
```

Comment:
Maybe we could implement a more extensive cleanup - maybe we could remove:
- Summon binary `/usr/local/bin/summon`
- Summon library `/usr/local/lib/summon/`
- Conjur configuration `/etc/conjur.conf`
- Conjur SSL cert Symlink `/etc/ssl/certs/conjur.crt`

Reply:
Done